README.md

```diff
@@ -476,7 +475,7 @@ Features
       TeamSpeak2 server statistics.
 
     - ted
-      Plugin to read values from `The Energy Detective' (TED).
+      Plugin to read values from `The Energy Detective` (TED).
```

Comment:
```suggestion
      Plugin to read values from "The Energy Detective" (TED).
```

Reply:
Everything else looks now fine (thanks for the extra items I had missed), but according to quick Googling, this **is** capitalized, not written as `"the energy detective" (ted)`?

Reply:
Will replace this with up to date comment.